package.json

```diff
@@ -46,9 +45,9 @@
     "eslint": "^5.6.1",
     "eslint-plugin-react": "^7.11.1",
     "husky": "^5.2.0",
-    "react": "16.9.0",
-    "react-dom": "16.9.0",
-    "react-native": "^0.61.4",
+    "react": "17.0.2",
```

Comment:
Thoughts on upgrades in the demo app as well?

Reply:
Yes, sure I will do it. @sshivananda